libs/velo-external-db-core/src/router.ts

```diff
@@ -94,173 +110,156 @@ export const createRouter = () => {
         res.send(appInfoPage)
     })
 
+    router.get('/capabilities', async(req, res) => {
+        const capabilitiesResponse = {
+            capabilities: {
+                collection: [capabilities.CollectionCapability.CREATE]
+            } as capabilities.Capabilities
+        } as capabilities.GetCapabilitiesResponse
+
+        res.json(capabilitiesResponse)
+    })
+
     router.post('/provision', async(req, res) => {
         const { type, vendor } = cfg
-        res.json({ type, vendor, protocolVersion: 2 })
+        res.json({ type, vendor, protocolVersion: 3 })
+    })
+
+    router.get('/id', async(req, res) => {
```

Comment:
This endpoint shouldn't be `/info` isn't it?
as written in the [profo file](https://github.com/wix-private/cloud/blob/cd837848a02f1faaa9931e06996e2b4ffe3ada93/data/cloud-data-spi/src/main/proto/com/wixpress/cloud/datasource/api/provisioning.proto#L17)